### ISSUE TITLE
Question about the causality Mask of SASRec

### ISSUE THREAD

**Issue by blueColdSea** [liuqidong07/LLM-ESR#4]
请问，我注意到SASRec 并没有加入 causality 的 attention，这是否正常？
代码位于models/SASRec.py
Excuse me, I noticed that SASRec did not include causality's attention. Is this normal?
The code is located in models/SASRec.py
`mha_outputs, _ = self.attention_layers[i](Q, seqs, seqs,) #   attn_mask=attention_mask)`

**Comment by ZhuYiZ-1205** [comment]
Thank you very much for the work you have done, I am reproducing your code and you have given a link to the preprocessed dataset at the end of ReadMe. I want to preprocess the dataset myself, can I just download the original dataset from the link given and follow the process for the Yelp dataset? For Beauty and Fashion dataset, I don't know which version to download, is it 2014 or 2018 or 2023 version? Should I download the 5-core or the ratings only? If you have downloaded the original dataset, can you pass me a copy, just for learning and exchange. Thank you very much!

Translated with DeepL.com (free version)

**Comment by ZhuYiZ-1205** [comment]
Thank you very much for the work you have done, I am reproducing your code and you have given a link to the preprocessed dataset at the end of ReadMe. I want to preprocess the dataset myself, can I just download the original dataset from the link given and follow the process for the Yelp dataset? For Beauty and Fashion dataset, I don't know which version to download, is it 2014 or 2018 or 2023 version? Should I download the 5-core or the ratings only? If you have downloaded the original dataset, can you pass me a copy, just for learning and exchange. Thank you very much!
